requirements.txt

```diff
@@ -16,12 +17,15 @@ django-rest-swagger==0.3.3
 django-summernote==0.6.8
 django-test-without-migrations==0.4
 djangorestframework==3.1.3
+git+https://github.com/celery/billiard.git#egg=billiard
```

Comment:
Can we pinpoint this to a specific version? When the package will be upgraded, it is not guaranteed that the same version as now will be installed.